README.md

```diff
@@ -85,7 +85,7 @@ hzcloud enterprise-cluster create \
   --cloud-provider=aws \
   --region=eu-west-2 \
   --zones=eu-west-2b \
-  --hazelcast-version=4.0 \
+  --hazelcast-version=5.0 \
```

Comment:
The current version is 5.0.2, so better to have --hazelcast-version=5.0.2. In the future, we will improve hz version selection smarter on backend side.